browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3170,9 +3173,12 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_WALLET_ETH_NAME" desc="Name of ETH cointype">
       Ethereum
     </message>
-    <message name="IDS_WALLET_SOL_NAME" desc="Name of SOL cointype">
+    <message name="IDS_WALLET_SOL_NAME" desc="Name of SOL cointype" translateable="false">
```

Comment:
let's do for Ethereum above as well, sorry didn't see it before :).

Reply:
No problem, adding it to Ethereum as well.